configure

```diff
@@ -3486,6 +3512,36 @@ if test -n "$_host"; then
 			_seq_midi=no
 			_timidity=no
 			;;
+		kos32)
+		        SDL_CFLAGS="-I${KOS32_SDK_DIR}/sources/SDL-1.2.2_newlib/include"
+			SDL_LIBS="-lSDLn -lsound"
+			_pkgconfig=/bin/false
+			_pkg_config=no
+
+			if test "$_freetype2" != no; then
+			    FREETYPE2_LIBS="-lfreetype.dll"
+			    FREETYPE2_STATIC_LIBS="-lfreetype.dll"
+			    # neither pkg-config nor freetype-config work, so we setup freetype manually
+			    _freetypepath="${KOS32_SDK_DIR}/sources/freetype/"
+			    FREETYPE2_CFLAGS="-I${KOS32_SDK_DIR}/sources/freetype/include"
+			    _freetype_found="true"
+			fi
+			if test "$_png" != no; then
+			    PNG_LIBS="-lpng16.dll -lz.dll"
+			    PNG_CFLAGS="-I${KOS32_SDK_DIR}/sources/libpng"
+			fi
+			if test "$_jpeg" != no; then
+			    JPEG_LIBS="-ljpeg.dll"
+			    JPEG_CFLAGS="-I${KOS32_SDK_DIR}/sources/libjpeg"
+			fi
+			_sdl=yes
+			_timidity=no
+			_sdlversion=1.2.2
+			_backend="kolibrios"
```

Comment:
Is there a reason for not simply using `_backend=sdl`?

Reply:
We need to override several things in backend, hence we make a derived backend